photutils/psf/psfphotometry.py

```diff
@@ -0,0 +1,342 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Module which provides classes to perform PSF Photometry"""
+
+from __future__ import division
+import abc
+import numpy as np
+from astropy.extern import six
+from astropy.table import Table
+from astropy.table import vstack
```

Comment:
Sorry, I didn't mean to have these in separate lines. Importing several packages in a line is not advised, while several submodules/functions from the same package is fine.

https://www.python.org/dev/peps/pep-0008/#imports